console/src/components/Services/Data/TableModify/ModifyActions.js

```diff
@@ -1889,12 +1505,12 @@ const saveColumnChangesSql = (colName, column, onSuccess) => {
     };
     const customOnError = () => {};
 
-    if (schemaChangesUp.length > 0) {
+    if (upMigration.length > 0 || downMigration.length > 0) {
```

Comment:
We don't want to call this if `upMigration` is empty and `downMigration` is not. `upMigration` needs to be there.